### ISSUE TITLE
[Updates] Should let the user update the SS.

### ISSUE THREAD

**Issue by NikolaiB** [subutai-io/base#171]
Improvement: Let the user an option to update SS. Would be nice if it shows the versions too. "You have Subutai 4.xx. New version is 5.x.x. Continue update? 'Y/n'

**Comment by Dilshat** [comment]
This issue is blocked by https://github.com/subutai-io/base/issues/167

**Comment by eabyshev** [comment]
duplicate of #812
